imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -144,16 +153,38 @@ class CaseExplorer extends Component {
   )
   filterCases = memoizeOne(
     (cases, searchText) => {
+      this.primeCaseId = null
       if (!searchText) return cases
-      let regex
 
-      // Checking the first character of the search term to determine if it's a numerical or alphabetical search
-      if (searchText.charAt(0).match(/\d/)) {
-        regex = new RegExp(`(^|[^\\d])${searchText}`, 'i') // Searching for a number starting as the search term
-      } else {
-        regex = new RegExp(`(^|[^a-zA-Z])${searchText}`, 'i') // Searching for a word starting as the search term
+      let caseResult, results, selectedCfg
+      let searchCfg = {
+        num: {
+          flds: ['_id', 'title', 'selectedUnit'],
+          redirectOnFieldMatch: '_id', // full match agains id will allow redirect to matched case uppon "Enter"
+          // regex: new RegExp(`(^|[^\\d])${searchText}`, 'i'),
+          regex: new RegExp(`${searchText}`, 'i')
```

Comment:
The simplified regex doesn't work for full word/ number matching like the original ones did. Can we restore them? I believe that provides a more useful search behavior